### ISSUE TITLE
Index entries cause stray spaces in text

### ISSUE THREAD

**Issue by wlievens** [RolfBremer/in-dexter#18]
This is my typst code:

```
#import "@preview/in-dexter:0.4.2": *

There is ($m/s$#index[meters-per-second]) a stray space before the closing parentheses

There is no ($m/s$) stray space before the closing parentheses
```

This is how it is rendered:
![image](https://github.com/user-attachments/assets/03c7f701-c199-430f-85f6-0f4a8ae4e084)

In the first line, which has an `#index` instruction, there is a stray space before the closing parenthesis.

**Comment by wlievens** [comment]
I just noticed it seems to be fixed if I put the `#index` instruction _inside_ the formula.
```
There is ($m/s#index[meters-per-second]$) a stray space before the closing parentheses
```
I'm not sure if this is a workaround or if it's supposed to work like this.

**Comment by RolfBremer** [comment]
This is a Bug in Version 0.4.2. It is already fixed in 0.4.3. here on GitHub. 
We will submit it to the Typst universe soon.

**Comment by wlievens** [comment]
Excellent, thank you.
